### PR TITLE
feat: publish as @stacks/connect

### LINKED ISSUE #624: Publish Connect under @stacks/connect
The Connect package should be published under @stacks/connect as discussed. References to Blockstack in code should be removed or replaced with Stacks.

### PR DESCRIPTION
**Pull request by yknl**
This PR renames Blockstack connect to Stacks connect and publish under Stacks org in NPM. Function `showBlockstackConnect` is renamed to `showConnect`.

Fixes #624

### CODE REVIEW

**Review by kyranjamie — COMMENTED** [review]
Maybe we can add a wrapper fn export of the same name so there are no breaking changes, using the `@deprecated` tsdoc comment.

```ts
/**
 * @deprecated Use the renamed `showConnect` method
 */
export const showBlockstackConnect = (authOptions: AuthOptions) => showConnect(authOptions);
```

**Review by kyranjamie — APPROVED** [review]

**Comment by hstove** [comment]
Because the Stencil refactor is ready to merge, I'll wait to merge this until after we merge the Stencil PR. Then, I can take on this work, as well as renaming the rest of the packages.

We'll need to also update dependencies for `@blockstack/connect` inside of the other projects in this repo. Also, we need to update docs to indicate these changes - which should be built on top of https://github.com/blockstack/docs/pull/764

**Pull request merged**
